yandex_contest/bin_sum.py

```diff
@@ -0,0 +1,53 @@
+def convert(string):
+    list_1 = []
+    list_1[:0] = string
+    list_x = [int(i) for i in list_1]
+    return list_x
```

Comment:
1) It's better to name functions in a way that it's clear what are they doing.
Eg, this one returns a list of integers based on an input argument, so it can be named accordingly.

2) Here you're creating a lot of unneeded objects. For example, you create an empty list. Then you fill the list with characters from the string in a quite unusual way. You can do it much simpler, eg, just 
```python3
list_1 = list(string)
``` 
will do absolutely the same thing

3) Then you're iterating through the list of characters and converting them to int -- and that means that you don't even need all the lists you created before. You can simply iterate through the `string` with the same for-loop:
```python3
list_x = [int(i) for i in string]
```

4) Then the whole function can be just a one-liner, so really you can even remove the function because it's quite clear what's happening. But it's ok to keep it if you think it's a bit more readable